RFS/src/main/java/com/rfs/cms/OpenSearchWorkCoordinator.java

```diff
@@ -123,7 +123,7 @@ public String toString() {
                             return "[ statusCode: " + r.getStatusCode() + ", payload: " + payloadStr + "]";
                         }
                     },
-                    (response, ignored) -> (response.getStatusCode() / 100) != 2);
+                    (response, ignored) -> (response.getStatusCode() / 100) == 2);
```

Comment:
This bug suggests there is a missing test case, can we add a unit test?

Reply:
agree - I hope to eventually pickup details like this once telemetry is in place.
Assert that the number of retries for this test run was >0, this one was 0, etc.

Reply:
Functionally, the code and tests converge as it was and is.
It doesn't do it efficiently.  I'd like to get a lot of efficiency numbers in place to confirm that we're working with an expected amount of contention, etc.  

I've opened https://opensearch.atlassian.net/jira/software/c/projects/MIGRATIONS/boards/1?selectedIssue=MIGRATIONS-1812 specifically to cover this case